File: daily_solution/Easy/1984. Minimum Difference Between Highest and Lowest of K Scores.py
class Solution:
    def minimumDifference(self, nums: List[int], k: int) -> int:
        nums.sort()
        result = float('inf')
        for i in range(len(nums)-k+1):
            temp = nums[i+k-1]-nums[i]
            result = min(temp, result)
        return result

File: daily_solution/Easy/1984. Minimum Difference Between Highest and Lowest of K Scores.java
import java.util.Arrays;
class Solution {
    public int minimumDifference(int[] nums, int k) {
        Arrays.sort(nums);
        int result = Integer.MAX_VALUE;
        for(int i =0;i<=nums.length-k;i++){
            int temp = nums[i+k-1]-nums[i];
            result = Math.min(temp,result);
        }
        return result;
    }
}
